net-analyzer/hexinject/hexinject-1.5.ebuild

```diff
@@ -0,0 +1,29 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+DESCRIPTION="Hexadecimal and raw packet injector and sniffer. Can be easily
```

Comment:
`DESCRIPTION` is way too long. If you want a long description, put it in `metadata.xml` (which you already have). This shouldn't be more than 80 chars.